tests/dummy/app/templates/application.hbs

```diff
@@ -1,17 +1,3 @@
 <h2 id="title">Welcome to Ember</h2>
 
-<p style="width: 362px; font: 16px sans-serif;">
-  {{truncate-multiline lines=1 text="supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious"}}
-</p>
-
-<p style="width: 362px; font: 16px sans-serif;">
-  {{#truncate-multiline lines=2 as |tm|}}
-    {{#tm.target}}
-      supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious
-    {{/tm.target}}
-    {{#tm.button}}
-      <span aria-hidden="true">{{if tm.isTruncated "⬇️" "⬆️"}}</span>
-      <span class="truncate-multiline--visually-hidden">{{if tm.isTruncated "view more text" "view less text"}}</span>
-    {{/tm.button}}
-  {{/truncate-multiline}}
-</p>
+{{outlet}}
```

Comment:
I think you probably upgraded with `ember init`, but we should probably put this back.